hard-protocol-home.js

```diff
@@ -255,15 +265,29 @@ const setApyDiplayValues = async (denoms, siteData, cssIds) => {
       const denomPrice = prices[rewardDenom].price;
       numerator += Number(reward[rewardDenom].amount) * denomPrice;
     }
-    const denomenator = balanceCurrency * collatDenomPrice;
+    const denominator = balanceCurrency * collatDenomPrice;
     let apy = '0.00%';
 
-    if (denomenator !==0) {
+    if (denominator !==0) {
       // use usdFormatter to truncate to 2 decimals and round
-      const apyWithDollarSign = usdFormatter.format((numerator / denomenator) * 100);
+      const apyWithDollarSign = usdFormatter.format((numerator / denominator) * 100);
       apy = formatPercentage(noDollarSign(apyWithDollarSign));
     }
-    const cssId = cssIds[denom]['apy'];
+    const cssId = cssIds[denom]['rapy'];
+    setDisplayValueById(cssId, apy)
+  }
+};
+
+const setSupplyApyDisplayValue = async (denoms, siteData, cssIds) => {
+  const interestRates = siteData['supplyApyByDenom'];
+  for (const denom of denoms) {
```

Comment:
this for loop with change a bit with the changes above, currently you iterate of denoms by doing `for (const denom of denoms)`, but you're not doing anything with the denom itself for the interest rate.  With the data formatted y denom for `siteData['interestRates']` you can do something like this:

`for (const denom of denoms) {
    // get interest rate if it exists, else set it to 0.00
    const apy = interestRates[denom] ? interestRates[denom] : '0.00'
    // get cssId
    const cssId = cssIds[denom]['sapy'];
    // format the apy value. usdFormatter adds commas and rounding
    const formattedAPY = formatPercentage(noDollarSign(usdFormatter.format(apy)))
    setDisplayValueById(cssId, formattedAPY)
}`